### ISSUE TITLE
Date ISO 8601 format check

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dorkasdev/bluecove#40]
```
In method com.intel.bluetooth.obexOBEXHeaderSetImpl.readTimeISO8601(byte
data[]) the expected date format is YYYYMMDDTHHMMSSZ. However, my Nokia
6500 slide phone returns date in local time zone, that means without final
'Z', and an exception is thrown. Would be nice if you extend the accepted
date format at least to that case, without trailing Z. Thank you!
```

Original issue reported on code.google.com by `Denis.Ku...@gmail.com` on 24 Jun 2008 at 2:09

**Comment by GoogleCodeExporter** [comment]
```
Was fixed in 2.0.3, March 06, 2008 
```

Original comment by `skarzhev...@gmail.com` on 24 Jun 2008 at 2:15
- Changed state: **Fixed-SNAPSHOT**
- Added labels: **OpSys-All**

**Comment by GoogleCodeExporter** [comment]
Original comment by `skarzhev...@gmail.com` on 24 Jun 2008 at 2:15

**Comment by GoogleCodeExporter** [comment]
```
Yes, I confirm it - the problem is solved in 2.0.3, thanks.
```

Original comment by `Denis.Ku...@gmail.com` on 1 Jul 2008 at 8:04

**Comment by GoogleCodeExporter** [comment]
Original comment by `skarzhev...@gmail.com` on 5 Sep 2008 at 7:15
- Changed state: **Fixed**
